Tests/conf.json

```diff
@@ -4042,9 +4042,9 @@
             "playbookID": "Trend Micro Deep Security - Test"
         },
         {
-            "integrations": "Carbon Black Endpoint Standard",
```

Comment:
please add another entry for the conf, and add the "Carbon Black Endpoint Standard" integration to skip if needed.

Reply:
Don't think it is a best practice delete a TPB.

Reply:
Done